pgcli/packages/sqlcompletion.py

```diff
@@ -20,20 +20,27 @@
 Special = namedtuple('Special', [])
 Database = namedtuple('Database', [])
 Schema = namedtuple('Schema', [])
+# AliasableTable is used in those cases we might want to generate an alias
+AliasableTable = namedtuple('AliasableTable', 'schema tables')
```

Comment:
Can we add a `aliasable_tables` field (default None) to Table, View, and Function instead of separate suggestion types?